install.ps1

```diff
@@ -0,0 +1,39 @@
+
+# Enable TLS 1.2 since it is required for connections to GitHub
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+
+# Determine latest Deno release via GitHub API
+$LatestRelease = Invoke-WebRequest `
+  -Uri "https://api.github.com/repos/denoland/deno/releases/latest" `
+    | ConvertFrom-Json `
+    | Select-Object -ExpandProperty "tag_name"
+
+# Create ~\.deno\bin directory if it doesn't already exist
+$DenoBin = "${Home}\.deno\bin"
```

Comment:
I think using `$var` instead of `${var}` to concatenate strings in PowerShell is usually an anti-pattern: If `$a = 'a'`, then `"$a:"` will throw: `Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.` Worse, `"$a:b"` will silently work and return an empty string. In this case, `$home` is not followed by a `:`, so everything is fine. I still decided to use `${var}` not just in this case but within other strings across the script due to its familiarity for JavaScript / TypeScript developers.